modules/actor/actor-wfrp4e.js

```diff
@@ -2586,10 +2587,10 @@ export default class ActorWfrp4e extends Actor {
     // If offhand and should apply offhand penalty (should apply offhand penalty = not parry, not defensive, and not twohanded)
     if (item.type == "weapon" && item.offhand.value && !item.twohanded.value && !(item.weaponGroup.value == "parry" && item.properties.qualities.defensive)) {
       modifier = -20
-      tooltip.push(game.i18n.localize("SHEET.Offhand"))
+      tooltip.push(`${game.i18n.localize("SHEET.Offhand")} (-20)`);
       modifier += Math.min(20, this.flags.ambi * 10)
       if (this.flags.ambi)
-        tooltip.push(game.i18n.localize("NAME.Ambi"))
+        tooltip.push(`${game.i18n.localize("NAME.Ambi")} (+${Math.min(20, this.flags.ambi * 10)})`);
```

Comment:
I've been fine with it elsewhere, but let's have a variable here, the `Math.min(20, this.flags.ambi * 10)` being duplicated bothers me 😅